src/api.ts

```diff
@@ -376,39 +400,29 @@ export function getCommitHashForBranch( branch: BranchName ): CommitHash | undef
 	return state.remoteBranches.get( branch );
 }
 
-export function touchCommit( hash: CommitHash ) {
-	state.accesses.set( hash, Date.now() );
+export function touchCommit( hash: CommitHash, env?: RunEnv ) {
+	const container = getRunningContainerForHash( hash, env );
+	if ( ! container ) throw `Running container for commit ${ hash } not found}`;
+
+	const name = getContainerName( container );
+	touchContainer( name );
 }
 
-export function getCommitAccessTime( hash: CommitHash ): number | undefined {
```

Comment:
As `state.accesses` now indexes usages by container name and not by commit hash, this method (and its tests) is not needed anymore. To achieve the same effect we must do `getContainerAccessTime( getContainerName( ... ))`.

This was only used once in `debug.tsx`, so I figured that having a wrapper for it was a bit of overengineering, but I can bring it back if we feel strongly about it.